arelle/plugin/validate/EDINET/Constants.py

```diff
@@ -3,6 +3,12 @@
 """
 from arelle.ModelValue import qname
 
+CORPORATE_DISCLOSURE_FORMS = frozenset([
+    "第二号の四様式",  # Form 2-4
+    "第二号の七様式",  # Form 2-7
+    "第三号様式",  # Form 3
+])
```

Comment:
This is fine for this PR, but there's a good chance we'll need to reference many/all form types, which may be a good use case for a form type `Enum` with something like `isCorporateDisclosureForm()`.